### PR TITLE
✨ [source-zendesk-support] Fix the `fields` data type to include the `id` and `value`

### PR DESCRIPTION
**Pull request by mohamagdy**
Set the JSON data type type of `fields` field items in Zendesk Support Tickets API

### CODE REVIEW

**Comment by marcosmarxm** [comment]
@mohamagdy please update the pull request description. Can you link the Zendesk documentation showing the field is valid? Did you run tests confirming the API actually return these values?

**Comment by mohamagdy** [comment]
@marcosmarxm thanks for taking a look into this PR. Unfortunately, I can't find the `fields` field in Zendesk documentation here https://developer.zendesk.com/api-reference/ticketing/tickets/tickets/#json-format. And yes, I can see the `fields` field returned from the API. What I noticed is that the values of the `custom_fields` and `fields` are the same (maybe this was for my case only). Maybe the `fields` and `custom_fields` are aliases, not sure 😕

**Comment by marcosmarxm** [comment]
@mohamagdy any chance to contact Zendesk to make clear this distinction? Would be quite helpful if you could this :)

**Comment by mohamagdy** [comment]
@marcosmarxm I asked Zendesk Customer Service and they pointed me to this page https://support.zendesk.com/hc/en-us/articles/4408886739098-About-ticket-fields saying that the `fields` field should contain the Standard Fields but the `custom_fields` should contain the Custom Fields.

File: airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/schemas/shared/tickets.json [hunk 1]
```diff
@@ -75,6 +75,17 @@
...
+      "items": {
+        "properties": {
+          "id": {
+            "type": ["null", "integer"]
+          },
+          "value": {
+            "type": ["null", "string"]
+          }
+        },
+        "type": ["null", "object"]
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/schemas/shared/tickets.json:87]:
```suggestion
        "type": ["null", "object"],
        "additionalProperties": true
```

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
Small change than ready to go.

File: airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/schemas/shared/tickets.json (hunk 1 above)
Comment (mohamagdy) [reply airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/schemas/shared/tickets.json]: Changed

**Review by lazebnyi — APPROVED** [review]
lgtm

After @marcosmarxm approve can be merged

**Review by marcosmarxm — APPROVED** [review]

**Pull request merged**